src/main/java/core/basesyntax/impl/StorageImpl.java

```diff
@@ -3,17 +3,54 @@
 import core.basesyntax.Storage;
 
 public class StorageImpl<K, V> implements Storage<K, V> {
+    private static final int INDEX_ZERO = 0;
+    private static final int STORAGE_LENGTH = 10;
+    private final StorageImpl<K, V>[] storage;
```

Comment:
Do not confuse two different things. StorageImpl - for storing a number of key-values. Create inner class Pair and use it for array
```suggestion
    private final Pair<K, V>[] storage;
```

Reply:
Don't understand. How can I implement this class? More important - why do I need to do it?